app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -293,6 +296,7 @@ class OverdueScreen : BaseScreen<
 
   override fun showNoOverduePatientsView() {
     viewForEmptyList.visibility = View.VISIBLE
+    if (isOverdueListDownloadAndShareEnabled) { buttonsFrame.visibility = View.GONE }
```

Comment:
Is this condition needed? 
We can just call this -
```suggestion
    buttonsFrame.visibility = View.GONE
```

Reply:
This feature is behind a feature flag, so keeping that for now. We would have to remove it feature flag, which can be done in a different PR.